Sources/LiUIComponents/Utils/LiHelperFunctions.swift

```diff
@@ -43,9 +41,9 @@ public class LiHelperFunctions {
         } else if hour >= 1 {
                 return "\(hour)" + LiHelperFunctions.localizedString(for:"h")
         } else if minute >= 1 {
-            return "\(minute) " + LiHelperFunctions.localizedString(for:"m")
+            return "\(minute)" + LiHelperFunctions.localizedString(for:"m")
         } else if second >= 3 {
-            return "\(second) " + LiHelperFunctions.localizedString(for:"s")
+            return "\(second)" + LiHelperFunctions.localizedString(for:"s")
         } else {
             return LiHelperFunctions.localizedString(for: "Just now")
         }
```

Comment:
Instead of `6h`, `1d`, etc would `6 hours ago`, `1 day ago`, etc be more appropriate?